force-app/main/default/lwc/miscRestApiCall/__tests__/miscRestApiCall.test.js

```diff
@@ -56,7 +56,7 @@ describe('c-misc-rest-api-call', () => {
         // Mock the successful fetch call. An empty response value is
         // sufficient, as we only test that fetch is called with the
         // expected parameter.
-        const fetch = (global.fetch = mockFetch({}));
+        const fetch = (global.fetch = mockFetch({ items: [] }));
```

Comment:
Nice one @albarivas. Thanks for going ahead and fixing this one. It's been on my list of little annoying things to fix.